### PR TITLE
GEOPY-259 : Use temp path for tests that rely on FlinFlon.geoh5

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-259 - Test accessing FlinFlon.geoh5 are failing on Github** 
 Attempted bug fix for failing tests that rely on FlinFlon.geoh5py.  
Fix involved creating a temporary .geoh5, copying entities over.

### CODE REVIEW

File: geoapps/utils/utils.py [hunk 1]
```diff
@@ -508,7 +508,7 @@ class LineDataDerivatives:
         padding.
     :param residual: Use the residual between the values and the running mean to compute derivatives.
     :param sampling: Sampling interval length (m) used in the FFT. Defaults to the mean data separation.
-    :param smoothing: Number of neighbours used by the :obj:`geoapps.utils.running_mean` routine.
+    :param smoothing: Number of neighbours used by the :obj:`geoapps.test_utils.running_mean` routine.
```
Comment (domfournier) [inline geoapps/utils/utils.py:511]: Was this intended?

**Review by domfournier — COMMENTED** [review]
Smart. Just have a look at that docstgring change

File: geoapps/utils/utils.py (hunk 1 above)
Comment (benk-mira) [reply geoapps/utils/utils.py]: Nope, I did a refactor->rename in pycharm and it borked a couple of names.  Just did a find and replace and pushed.  Will see if the tests pass now.

**Review by domfournier — DISMISSED** [review]
Looking good. We will need to update the rest of the tests along those lines at some point.

**Review by domfournier — APPROVED** [review]
Great, just waiting for geoh5py

**Review by domfournier — APPROVED** [review]
Weird ... approved again.

**Pull request merged**